marathon/tests/test_integration.py

```diff
@@ -0,0 +1,18 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+from copy import deepcopy
+
+from . import common
+
+pytestmark = pytest.mark.integration
```

Comment:
Looks unused? Is that a tricky to mark the whole module?

Reply:
Yup